erpnext/support/doctype/service_level/test_service_level.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+from __future__ import unicode_literals
+from erpnext.hr.doctype.employee_group.test_employee_group import make_employee_group
+from erpnext.hr.doctype.holiday_list.test_holiday_list import make_holiday_list
+
+import frappe
+import unittest
+
+class TestServiceLevel(unittest.TestCase):
+
+	def test_service_level(self):
```

Comment:
@hrwX These tests are not useful. Here you are checking if you are getting the inserted data... why do we want to test that!

We need tests to check the SLA functionalities with different cases. 

eg.test to check Response time allocation if there's a holiday in between, or test to check if the status is changed accordingly based on the response time, or test check if only working hours are considered while calculating the response time, etc.

Reply:
Have checked the test functionalities in DocType Issue tests

Reply:
@hrwX Yes, but still those tests are bit weak and are of the similar kind... need to cover more scenarios